assets/src/styles/blocks/Cookies.scss

```diff
@@ -6,4 +6,36 @@
       font-size: $font-size-md;
     }
   }
+
+  .cookies-checkbox-description.rich-text {
```

Comment:
Shouldn't this be for the editor only?

Reply:
Indeed, but this block doesn't have split files yet

Reply:
We can just add the file in this PR, it's an easy way to improve front end perofrmance a bit.

Reply:
Good point! Updated 🙂